js/timers.ts

```diff
@@ -181,7 +185,15 @@ function setTimer(
   // and INT32_MAX. Any other value will cause the timer to fire immediately.
   // We emulate this behavior.
   const now = getTime();
+  if (delay > TIMEOUT_MAX) {
+    console.warn(
+      `${delay} does not fit into a 32-bit signed integer.
+Timeout duration was set to 1.`
```

Comment:
Please make it so there's not a line break in the template literal.

Reply:
@ry Thanks 👍fixed.